lib/dbservice_web/controllers/group_controller.ex

```diff
@@ -34,6 +59,42 @@ defmodule DbserviceWeb.GroupController do
             }
           })
         end,
+      GroupSession:
```

Comment:
shouldn't this be `GroupSessions`? As you're returning multiple sessions in the API response?